python/src/otel/Makefile

```diff
@@ -3,7 +3,7 @@ export SDK=$(shell pwd)/otel_sdk
 build-OTelLayer:
 	mkdir -p $(ARTIFACTS_DIR)/python
 	python3 -m pip install -r $(SDK)/requirements.txt -t $(ARTIFACTS_DIR)/python
-	python3 -m pip install -r $(SDK)/requirements-nodeps.txt -t $(ARTIFACTS_DIR)/tmp --no-deps
+	python3 -m pip install -r $(SDK)/nodeps-requirements.txt -t $(ARTIFACTS_DIR)/tmp --no-deps
```

Comment:
haha, oops... turns out dependabot was already working with the previous name. I could revert this part of the change or just leave it.